torch/nn/quantized/modules/rnn.py

```diff
@@ -0,0 +1,288 @@
+import numbers
+from typing import Optional, Tuple
+
+import torch
+from torch import Tensor
+import torch.nn as nn
+
+from .functional_modules import FloatFunctional
+
+
+"""
+We will recreate all the RNN modules as we require the modules to be decomposed
+into its building blocks to be able to observe.
+"""
+
+def _hidden_as_output(x):
+    if isinstance(x, Tensor):
+        return x
+    return x[0]
+
+class LSTMCell(torch.nn.Module):
```

Comment:
Since these are fp32 modules, lets have them in a different name-space. torch/nn/quantizable? (cc@vkuzo)